builtin/providers/aws/resource_aws_appautoscaling_target.go

```diff
@@ -60,16 +41,16 @@ func resourceAwsAppautoscalingTarget() *schema.Resource {
 				ForceNew: true,
 			},
 			"scalable_dimension": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  "ecs:service:DesiredCount",
-				ForceNew: true,
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateAppautoscalingScalableDimension,
 			},
 			"service_namespace": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  "ecs",
-				ForceNew: true,
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
Are the defaults in place from pre this PR correct here? Otherwise we are breaking backwards compatibility